Tasks/Gradle/Gradle.ps1

```diff
@@ -90,12 +90,22 @@ if ($jdkPath)
     Write-Verbose "JAVA_HOME set to $env:JAVA_HOME"
 }
 
+if($codeCoverageTool.equals("Jacoco"))
+{
+    $summaryFileName = "summary.xml"
+    $reportingTaskName = "jacocoTestReport"
+}
```

Comment:
check the comment below. if it's single module "jacocoTestReport". if it's multi module "jacocoRootReport"